src/experiments/permutations.spec.ts

```diff
@@ -1,5 +1,7 @@
-import assert from 'assert';
-import { describe, it, beforeEach, afterEach, before } from 'mocha';
+/**
+ * @jest-environment node
+ */
+
```

Comment:
Why did we add this? What is the logic behind it?

Reply:
Basically jest requires you to set an environment (`node`, `browser`, etc.) so that it better understands how to test things.

This is usually done at the config level, using multiple projects and config files. See more info here: https://stackoverflow.com/questions/41318115/testing-two-environments-with-jest.

The comment block above allows us to specify in a file the environment it should be tested in. It is more of a temporary thing, to be able to ship this PR.

I'll create a ticket to set this up a bit better later.

Works for you @iliapir2?

Reply:
https://github.com/Sheertex/next-experiments/issues/8

Reply:
Yep, that works!